components/PlayerTable.tsx

```diff
@@ -4,13 +4,15 @@ import styled from 'styled-components';
 
 // import Skeleton, { SkeletonTheme } from 'react-loading-skeleton';
 // import Link from '../../components/LinkWithSeason';
-import { Player, Goalie, SearchType } from '../..';
```

Comment:
The main gist of all these changes in this file is that we had a lot of repeated code.  I generalized the ratings and stats table to use only one table that takes different parameters.  This makes sure all our tables only have to be written once if we want to make a change.